shiny/test/_pytest.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+from pathlib import PurePath
+from typing import Generator
+
+import pytest
+
+from ._conftest import ShinyAppProc
+from ._playwright import BrowserContext, Page
+from .fixture import local_app_fixture_gen
+
+
+# Make a single page fixture that can be used by all tests
+@pytest.fixture(scope="session")
```

Comment:
Can we move `session_page` and `page` into shiny's conftest.py? These two fixtures feel like they are too shiny specific for the first release